baseapp-wagtail/baseapp_wagtail/settings.py

```diff
@@ -0,0 +1,75 @@
+from baseapp_core.settings.env import env
+
+WAGTAIL_INSTALLED_APPS = [
+    # wagtail-headless-preview
+    "wagtail_headless_preview",
+    # wagtail
+    "wagtail.api.v2",
+    "wagtail.contrib.settings",
+    "wagtail.contrib.redirects",
+    "wagtail.embeds",
+    "wagtail.sites",
+    "wagtail.snippets",
+    "wagtail.documents",
+    "wagtail.images",
+    "wagtail.search",
+    "wagtail.admin",
+    "wagtail.locales",
+    "wagtail.users",
+    "wagtail",
+    # wagtail dependencies
+    "modelcluster",
+    "taggit",
+]
+
+WAGTAIL_INSTALLED_INTERNAL_APPS = [
+    "baseapp_wagtail",
+    "baseapp_wagtail.base",
+    "baseapp_wagtail.medias",
+]
+
+WAGTAIL_MIDDLEWARE = [
+    "wagtail.contrib.redirects.middleware.RedirectMiddleware",
+]
+
+# Locale settings
+LANGUAGE_CODE = "en-us"
+TIME_ZONE = "UTC"
+USE_I18N = True
+USE_L10N = True
+USE_TZ = True
+
+WAGTAIL_CONTENT_LANGUAGES = LANGUAGES = [
+    ("en", "English"),
+]
+
+# Wagtail settings
+WAGTAILADMIN_BASE_URL = env("URL", default="http://localhost:8000")
+
+# Wagtail Preview
+FRONT_HEADLESS_URL = env("FRONT_HEADLESS_URL", default="http://localhost:3000/pages")
```

Comment:
can we reuse `settings.FRONT_URL` here?

Reply:
Not really. FRONT_URL is used by the baseapp packages and it points to the FE domain. The FRONT_HEADLESS_URL points to where the wagtail pages will be rendered. As the default behavior of the wagtail package, the wagtail pages will be rendered under the `/pages/` path. That's why we need a var only for that :)

Reply:
@Hercilio1 are we expecting to have a different frontend for it? Maybe we can just append `/pages/` to `FRONT_URL`, what you think?

Reply:
We would need an var for that suffix then. Something like:
```
FRONT_URL = 'http://localhost:3000'
WAGTAIL_FRONT_URL_PATH = '/pages'
```
What do you think, @nossila ?

Reply:
I like that idea @Hercilio1 we could also just use FRONT_URL and append the path to it

Reply:
@nossila, the `FRONT_URL` is used by other packages. The `/pages` suffix is specific to Wagtail. Can we keep it separate like that?

Reply:
I like this idea.